tests/loadtests/vkloadtests/VulkanLoadTestSample.h

```diff
@@ -60,7 +60,7 @@ class VulkanLoadTestSample : public LoadTestSample {
     vk::PipelineShaderStageCreateInfo
     loadShader(std::string filename,
                vk::ShaderStageFlagBits stage,
-               std::string modname = "main");
+               const char* = "main");
```

Comment:
Why are you changing this?

Reply:
Because the string is used in the return value so it's undefined behaviour.